erpnext/accounts/doctype/coupon_code/coupon_code.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+from frappe.model.document import Document
+
+class CouponCode(Document):
+	def autoname(self):
```

Comment:
explicit is better than implicit. The hardcoding of name should be avoided

Reply:
Thanks! it coupon_code is made explicit.